src/components/common/markets_dropdown.tsx

```diff
@@ -441,4 +436,38 @@ export class MarketsDropdown extends React.Component<Props, State> {
 
         return <DayChange>{percentChange}%</DayChange>;
     };
+
+    private readonly _getDayVolumen: any = (item: Token) => {
+        const tokenDummy = MARKETS_LIST.find(obj => {
+            return obj.symbol === item.symbol;
+        });
+        return tokenDummy ? tokenDummy.dayVol : '';
+    };
+
+    private readonly _getPrice: any = (item: Token) => {
+        const tokenDummy = MARKETS_LIST.find(obj => {
+            return obj.symbol === item.symbol;
+        });
+        return tokenDummy ? tokenDummy.price : '';
+    };
 }
+
+const mapStateToProps = (state: StoreState): PropsToken => {
+    return {
+        tokens: getTokens(state),
+        selectedToken: getSelectedToken(state),
+    };
+};
+
+const mapDispatchToProps = (dispatch: any) => {
+    return {
+        setSelectedToken: (token: Token) => dispatch(setSelectedToken(token)),
```

Comment:
We should dispatch the _orderbook_ and _own orders_ actions here too.

I'm not sure if we should create a unique thunk-action that does the 3 things in one: update selected token + get order book + get own actions.
Any thoughts on this @fvictorio ?

Reply:
@unjapones Done!